### PR TITLE
DOCSUP-14784: Document the system.session_log system table

### PR DESCRIPTION
**Pull request by sevirov**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:

- Documentation for #22415

### CODE REVIEW

File: docs/en/operations/system-tables/session_log.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+Contains information about all successful and failed login and logout events.
+
+Columns:
+
+-   `type` ([Enum8](../../sql-reference/data-types/enum.md)) — Login/logout result. Possible values:
+    -   `LoginFailure` — Login error.
+    -   `LoginSuccess` — Successful login.
+    -   `Logout` — Logout from the system.
+-   `session_id` ([UUID](../../sql-reference/data-types/uuid.md)) — Session id.
+-   `session_name` ([String](../../sql-reference/data-types/string.md)) — Session name.
```
Comment (vitlibar) [inline docs/en/operations/system-tables/session_log.md:12]: The columns `session_id`, `session_name`, and `changed_settings` have changed, see https://github.com/ClickHouse/ClickHouse/pull/30882

Comment (sevirov) [reply docs/en/operations/system-tables/session_log.md]: @vitlibar Ok. I renamed the columns.

File: docs/en/operations/system-tables/session_log.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+Contains information about all successful and failed login and logout events.
+
+Columns:
+
+-   `type` ([Enum8](../../sql-reference/data-types/enum.md)) — Login/logout result. Possible values:
+    -   `LoginFailure` — Login error.
+    -   `LoginSuccess` — Successful login.
+    -   `Logout` — Logout from the system.
+-   `auth_id` ([UUID](../../sql-reference/data-types/uuid.md)) — Session id.
+-   `session_id` ([String](../../sql-reference/data-types/string.md)) — Session name.
```
Comment (vitlibar) [inline docs/en/operations/system-tables/session_log.md:12]: Wrong. `auth_id` is not a `session id`, they're different things! Session ID is passed by client via HTTP interface. Please write that and also put here a link to the explanation in https://clickhouse.com/docs/en/interfaces/http/

`auth_id` is authentication ID, it's an UUID which is generated automatically each time a user logins.

Comment (sevirov) [reply docs/en/operations/system-tables/session_log.md]: > Wrong. `auth_id` is not the same as `session_id`, they're different things! Session ID is passed by client via HTTP interface. Please write that and also put here a link to the explanation in https://clickhouse.com/docs/en/interfaces/http/
> 
> `auth_id` is authentication ID, it's an UUID which is generated automatically each time a user logins.

@vitlibar Thank you. I fixed.

**Review by vitlibar — APPROVED** [review]

**Pull request merged**